chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/QuestionableActivityEmailJob.java

```diff
@@ -88,9 +93,15 @@ public void execute(final JobExecutionContext jobContext) throws JobExecutionExc
         LOGGER.info("********* Starting the Questionable Activity Email job. *********");
         LOGGER.info("Creating questionable activity email for: " + jobContext.getMergedJobDataMap().getString("email"));
 
+        Integer range = getRangeInDays(jobContext);
+        if (!isRangeValid(range)) {
+            LOGGER.error(
+                    "Range is invalid.  It must be between " + MIN_RANGE_IN_DAYS + " and " + MAX_RANGE_IN_DAYS + ".");
+            return;
```

Comment:
Thinking about the user experience, if someone managed to create a job with an invalid range there's no way they'd ever know it, right? As in, when the job starts it'd have an invalid range and would then just end, without sending any message. Would it make sense to default to the DEFAULT_RANGE value and maybe even add a message to the email about that? We're defaulting them if they don't have a value at all - it might be reasonable to do the same thing if they have an invalid one